docs/api/tracker-stores.rst

```diff
@@ -70,6 +70,63 @@ SQLTrackerStore
     - ``login_db`` (default: ``None``): Alternative database name to which initially  connect, and create the database specified by `db` (PostgreSQL only)
     - ``query`` (default: ``None``): Dictionary of options to be passed to the dialect and/or the DBAPI upon connect
 
+
+:Supported Databases:
+    - PostgreSQL
+    - Oracle > 11.0
+    - SQLite
+
+
+:Oracle Configuration:
+
+
+      To use the SQLTrackerStore with Oracle, there are a few additional steps.
+      First, create a database ``tracker`` in your Oracle database and create a user with access to it.
+      Create a sequence in the database with the following command (where username is the user you created):
```

Comment:
optional: add link to oracle docs: https://docs.oracle.com/cd/B28359_01/server.111/b28286/statements_6015.htm

Reply:
or maybe better: https://docs.oracle.com/cd/B28359_01/server.111/b28310/views002.htm#ADMIN11792